src/utils/testing.ts

```diff
@@ -0,0 +1,40 @@
+import type { ValueSetterParams } from 'ag-grid-community'
+
+import type { Student } from '@/services/students'
+
+export const ensureGridApiHasBeenSet = (vm: any) =>
+  new Promise<void>(function (resolve) {
+    ;(function waitForGridReady() {
```

Comment:
is it something that i'm not aware of or just mistake? :D

Reply:
I follow this: https://www.ag-grid.com/vue-data-grid/testing/#waiting-for-the-grid-to-be-initialised

Without this gridApi will be undefined in tests